subprojects/test-runner/service/src/test/kotlin/com/avito/runner/service/DeviceWorkerTest.kt

```diff
@@ -124,6 +127,87 @@ class DeviceWorkerTest {
                 .isEqualTo(intentions)
         }
 
+    @Test
+    fun `onTestFinished should be called worker run`() =
```

Comment:
Let's divide explicitly [differents parts](https://avito-tech.github.io/avito-android/contributing/Testing/#naming-conventions) in the name